### PR TITLE
Fix/memory leaks

### PR DESCRIPTION
**Pull request by andycall**
Fixed https://github.com/openwebf/webf/issues/482

Fixed the memory leaks caused by the UICommand::disposedBindingObject command not being executed. 

These memory leaks affected versions 0.14.1 to 0.15.0 and 0.13.3

### CODE REVIEW

File: webf/lib/src/bridge/to_native.dart [hunk 1]
```diff
@@ -582,3 +602,27 @@ void flushUICommand(WebFViewController view) {
...
+  List<UICommand> commands = readNativeUICommandToDart(nativeCommandItems, commandLength, disposeUICommand: true);
+
+  for (int i = 0; i < commandLength; i++) {
+    UICommand command = commands[i];
+    UICommandType commandType = command.type;
+    Pointer nativePtr = command.nativePtr;
+
+    if (commandType == UICommandType.disposeBindingObject) {
+      BindingObject? bindingObject = BindingBridge.getBindingObject(nativePtr);
+      print('dispose $nativePtr');
```
Comment (XGHeaven) [inline webf/lib/src/bridge/to_native.dart:623]: remove print

**Review by XGHeaven — CHANGES_REQUESTED** [review]
And the better to change the title to a good one.

**Comment by zjt123** [comment]
这个是否能弄个测试用例，便于自测及时发现问题

**Review by XGHeaven — APPROVED** [review]

**Comment by andycall** [comment]
@zjt123 added

**Review by zjt123 — APPROVED** [review]

**Pull request merged**